recipes/xtensor-zarr/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -e

```
just to make sure errors are caught